Rocket.Chat/Theme/ThemeableViews.swift

```diff
@@ -333,11 +334,22 @@ extension UIPickerView {
 
 // MARK: External class extensions
 
-extension SLKTextInputbar {
+extension HintsView {
+    override func applyTheme() {
+        super.applyTheme()
+        guard let theme = theme else { return }
+        backgroundColor = theme.backgroundColor
```

Comment:
I think, we don't require this line. The background color is automatically set to `theme.backgroundColor` when `super.applyTheme` is called.

Reply:
:+1: